sparkpost/django/email_backend.py

```diff
@@ -0,0 +1,79 @@
+from django.core.mail.backends.base import BaseEmailBackend
+from sparkpost import SparkPost
+from django.conf import settings
+from .exceptions import UnSupportedContent, UnSupportedParam
+
+
+class SparkPostEmailBackend(BaseEmailBackend):
+    """
+    SparkPost wrapper for Django email backend
+    """
+
+    def __init__(self, fail_silently=False, **kwargs):
+        super(SparkPostEmailBackend, self)\
+            .__init__(fail_silently=fail_silently, **kwargs)
+
+        sp_password = getattr(settings, 'SP_PASSWORD', None)
```

Comment:
Yeah but this is actually using the SparkPost API under the hood and there isn't really a concept of a password there. It's all in how we document usage.